### PR TITLE
Date fixes

### PR DESCRIPTION
**Pull request by mattsiel**
# Description

This PR includes the following proposed change(s):

- Fixed the date of the licensees being display incorrectly
- Fixed the renewal date of the software being display incorrectly
- moved the logo
- 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/common/DateCell.jsx [hunk 1]
```diff
@@ -5,7 +5,14 @@ import { useEffect, useState } from 'react';
 const DAYS_BEFORE_EXPIRATION_WARNING = 90;
 
 export const DateCell = ({ value }) => {
-  const date = DateTime.fromISO(value);
+  console.log(value);
```
Comment (dbarkowsky) [inline src/components/common/DateCell.jsx:8]: Console logs should probably come out.
One on line 14 as well.

File: src/components/formik/inputs/LicenseDropdownControl.jsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const LicenseDropdownControl = (props) => {
 
   return (
     <span style={remainingStyle()}>
-      {`${title}   (${used} remaining out of ${total})`}
+      {`${title}   (${used} assigned of ${total})`}
```
Comment (dbarkowsky) [inline src/components/formik/inputs/LicenseDropdownControl.jsx:30]: Will's comment in the ticket says 
> The difference between the Quantity and the sum of the assigned licenses should be displayed in the list of Titles that have not yet been added to a Licensee.

So I think they're looking for something like 
```javascript
{`${title} (${total - used} remaining)`}
```

**Review by dbarkowsky — COMMENTED** [review]
Somehow I'm getting the previous date now. Entered 2033-03-03 at 3:32pm system time. Date was displayed as 2033-03-02.

File: src/components/formik/inputs/LicenseDropdownControl.jsx (hunk 2 above)
Comment (mattsiel) [reply src/components/formik/inputs/LicenseDropdownControl.jsx:30]: I think he wanted to use the work assigned, the ticket has an extra comment about that.

File: src/components/common/DateCell.jsx (hunk 1 above)
Comment (mattsiel) [reply src/components/common/DateCell.jsx:8]: thanks for catching them

File: src/components/common/ModifiedCell.jsx [hunk 3]
```diff
@@ -9,8 +9,8 @@ const DAYS = 28;
 
 export const ModifiedCell = ({ value }) => {
   // Date is changed from UTC to PST then formated to ISO
-  const modifiedDate = value.row.modified; // .replace('Z', '+07:00');
-  const date = DateTime.fromISO(modifiedDate);
+  // .replace('Z', '+07:00');
+  const date = DateTime.fromISO(value.row.modified);
```
Comment (dbarkowsky) [inline src/components/common/ModifiedCell.jsx:13]: More clean than the original!

**Review by dbarkowsky — APPROVED** [review]
Fixes appear to be working.

**Pull request merged**